exploration-module/src/main/java/io/edpn/backend/exploration/configuration/ServiceConfig.java

```diff
@@ -0,0 +1,7 @@
+package io.edpn.backend.exploration.configuration;
+
+import org.springframework.context.annotation.Configuration;
+
+@Configuration("ExplorationModuleServiceConfig")
+public class ServiceConfig {
+}
```

Comment:
Do we need empty config classes? Or do they exist only because it standardises the approach?

Granted they're singletons, but each class created adds a memory overhead.